processing/run.sh

```diff
@@ -9,6 +9,8 @@ export OSM_FILTERED_FILE=${OSM_DATADIR}openstreetmap-filtered.osm.pbf
 # define default args
 export SKIP_DOWNLOAD=${SKIP_DOWNLOAD:-0}
 export SKIP_TAG_FILTER=${SKIP_TAG_FILTER:-0}
+export COMPUTE_DIFFS=${COMPUTE_DIFFS:-0}
+export FREEZE_DATA=${FREEZE_DATA:-0}
```

Comment:
@rush42 we need to add this to https://github.com/FixMyBerlin/atlas-geo/blob/91abab8dd4ec50e46b4e2403494f3ad71db3df6f/docker-compose.yml#L5-L20 as well I think. Or at least we should, so we have a consistent way of handling the .env variables in docker.

Reply:
Ich habe die Variable dort hinzugefügt. Meinst du, dass wir dort auch die default Argumente setzen sollten ?

Reply:
@rush42 habe ich keine Meinung zu / kenne ich mich nicht genug mit aus. Aber für mich hat es etwas mit https://github.com/FixMyBerlin/private-issues/issues/1643#issuecomment-2090842692 zu tun. Ich habe diesen Kommentar mal dort referenziert.